### ISSUE TITLE
PHP warning: array given to htmlspecialchars()

### ISSUE THREAD

**Issue by LarsGit223** [flack/UniversalFeedCreator#7]
Hi,
I am forwarding this issue from a DokuWiki user who noticed this warning:

```PHP Warning:  htmlspecialchars() expects parameter 1 to be string, array given in .../vendor/openpsa/universalfeedcreator/lib/Creator/AtomCreator10.php on line 113```

I don't have any further information yet.

Greetings,
Lars

**Comment by flack** [comment]
At first glance it looks like this is happening because one of the feed items has multiple categories. I have to say, I'm not overly familiar with the Atom spec, do you know if this is valid? if so, we should probably simply do the same as here:

https://github.com/flack/UniversalFeedCreator/blob/master/lib/Creator/RSSCreator091.php#L154-L162

**Comment by LarsGit223** [comment]
No, I don't know but it looks like it may have multiple categories:

> category Specifies a category that the feed belongs to.  A feed may have multiple category elements.  More info here. <category term="sports"/>

Taken from https://validator.w3.org/feed/docs/atom.html, see section "Optional feed elements".

**Comment by flack** [comment]
Should be fixed now

**Comment by LarsGit223** [comment]
Nice. Thanks :+1:
